server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/rfc8621/RFC8621MethodsModule.java

```diff
@@ -198,7 +197,6 @@ protected void configure() {
         blobResolverMultibinder.addBinding().to(MessageBlobResolver.class);
         blobResolverMultibinder.addBinding().to(UploadResolver.class);
         blobResolverMultibinder.addBinding().to(MessagePartBlobResolver.class);
-        blobResolverMultibinder.addBinding().to(AttachmentBlobResolver.class);
```

Comment:
Q: if it is unused, why dont we remove the class code also?

Reply:
Asked:
anyone knows if AttachmentBlobResolver is still in use or not? It seems that tests still pass without it

Benoit:
Not used.
But keep it arround.
25% salary raise to the genius that would make attachment download use it in JMAP RFC-8621

50% salary raise if temporal coupling is avoided and the solution not too hacky